packages/manager/modules/pci/src/projects/project/storages/databases/components/tags-input/tags-input.controller.js

```diff
@@ -8,6 +8,9 @@ export default class PciTagsInputController {
     this.model.forEach((element) => {
       this.items.push({ title: element });
     });
+    this.min = this.min || 1;
+    this.max = this.max || 255;
```

Comment:
can you move the constants to constants file or define as a constant at the top of the file

Reply:
Fixed here: https://github.com/ovh/manager/pull/5627/commits/d4c735fd418d97af581a0b760c05fd8825501442